### ISSUE TITLE
Feature request : Goals, like on Kickstarter

### ISSUE THREAD

**Issue by gabwiel** [itchio/itch.io#929]
It would be awesome to be able to set goals for our games

**Comment by fasterthanlime** [comment]
You can create sales and set goals for those! See https://itch.io/sales/with-goal

**Comment by gabwiel** [comment]
No not that type of goal, I mean multiple goals to add new features to the game, like :

- 100€ : New World
- 250€ : Single Player mode
...

**Comment by SkyTech6** [comment]
You could connect your project to Patreon with the already built in integration there. Then use the Patreon Goals system in the same fashion (as well giving people Discord perks to discuss the game they're supporting also!)

**Comment by gabwiel** [comment]
Yeah but I don't want to make a patreon account, i'd like to do that on itch.io

**Comment by leafo** [comment]
Have you considered writing your goals into the description of the page, and then updating the goal of the sale as you cross different thresholds? As far as I know, Kickstarter doesn't have goals either, and most creators have some sort of graphic that they update.
